lib/express-handlebars.js

```diff
@@ -328,7 +328,7 @@ ExpressHandlebars.prototype._resolveLayoutPath = function (layoutPath) {
         layoutPath += this.extname;
     }
 
-    if (layoutPath.charAt(0) !== '/') {
+    if (layoutPath.charAt(0) !== '/' && (':' !== layoutPath.charAt(1) || '\\' !== layoutPath.charAt(2))) {
```

Comment:
I think it can be simpler and more robust to simply remove this conditional and replace it with:

```js
return path.resolve(this.layoutsDir, layoutPath);
```

But we'd need to test it to see if it does the right thing.